sui-execution/latest/sui-adapter/src/static_programmable_transactions/typing/verify/memory_safety.rs

```diff
@@ -368,13 +368,13 @@ fn move_value(
     if context.is_location_borrowed(l)? {
         // TODO more specific error
         return Err(command_argument_error(
-            CommandArgumentError::InvalidValueUsage,
+            CommandArgumentError::CannotMoveBorrowedValue,
```

Comment:
I wanted more granular tests... but we should also just give better error cases to users :)